src/inputs/Table/components/ImportButton/ImportButton.js

```diff
@@ -3,26 +3,32 @@
 
 import React from 'react';
 import UploadFileIcon from '@mui/icons-material/UploadFile';
-import { IconButton } from '@mui/material';
+import { CircularProgress, IconButton } from '@mui/material';
 import PropTypes from 'prop-types';
 import { FadingTooltip } from '../../../../misc';
 
 export const ImportButton = (props) => {
-  const { disabled, onImport, label } = props;
+  const { onImport, isLoading, disabled, label } = props;
   return (
     <FadingTooltip title={label} useSpan={true}>
-      <IconButton
-        variant="text"
-        component="label"
-        onChange={onImport}
-        size="small"
-        color="primary"
-        disabled={disabled}
-        data-cy="import-file-button"
-      >
-        <input type="file" accept=".csv, .xlsx" hidden />
-        <UploadFileIcon fontSize="inherit" />
-      </IconButton>
+      {isLoading ? (
+        <IconButton variant="text" component="label" size="small" disabled>
+          <CircularProgress disableShrink color="primary" size="15px" />
+        </IconButton>
```

Comment:
Instead of duplicating the code of the loading spinner in all buttons, we should factorize it to make the code more maintainable :slightly_smiling_face: 
Furthermore, if you use the same value of `isLoading` in all buttons, you will have 3 spinners visible at the same time.